src/helpers/uniqueAssetTraitDisplayTypeCompareFunction.ts

```diff
@@ -0,0 +1,33 @@
+import { UniqueAsset } from '@rainbow-me/entities';
+
+type Trait = UniqueAsset['traits'][number];
+type ReturnType = -1 | 0 | 1;
+
+// lower number = higher precedence
+const displayTypeRanks = new Map<string | undefined | null, number>([
+  [undefined, 0],
+  [null, 0],
+  ['number', 0],
+  ['boost_number', 1],
+  ['boost_percentage', 1],
+  ['date', 2],
+]);
```

Comment:
```suggestion
const displayTypeRanks =  {
  'boost_number': 1,
  'boost_percentage': 2,
  'date': 3,
} as const;
``` I think we can keep this simple and just default to 0. Also, I prefer `boost_percentage` after `boost_number